templates/docs/patterns/search-and-filter.md

```diff
@@ -6,8 +6,6 @@ context:
 
 ## Search and filter
 
-<span class="p-label--new">New</span>
```

Comment:
That's why I was usually keeping those for a while - we just added this component. It's still 'new', even when we soon release new version.

Hard to say how long we should keep those. Would be nice to have it automated in some way...